dirsrvtests/tests/suites/clu/dsidm_account_test.py

```diff
@@ -0,0 +1,148 @@
+#
+import time
+import subprocess
+import pytest
+import logging
+import os
+
+from lib389 import DEFAULT_SUFFIX
+from lib389.cli_idm.account import list, get_dn, lock, unlock, delete, modify, rename, entry_status, \
+    subtree_status, reset_password, change_password
+from lib389.topologies import topology_st
+from lib389.cli_base import FakeArgs
+from lib389.utils import ds_is_older
+from lib389.idm.user import nsUserAccounts
+
+pytestmark = pytest.mark.tier0
+
+LOG_FILE = '/tmp/dsidm_account.log'
+logging.getLogger(__name__).setLevel(logging.DEBUG)
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def set_log_file(request):
+    fh = logging.FileHandler(LOG_FILE)
+    fh.setLevel(logging.DEBUG)
+    log.addHandler(fh)
+
+    def fin():
+        log.info('Delete log file')
+        os.remove(LOG_FILE)
+
+    request.addfinalizer(fin)
+
+
+def check_value_in_log_and_reset(content_list=None, content_list2=None, check_value=None, check_value_not=None):
+    with open(LOG_FILE, 'r+') as f:
+        file_content = f.read()
+        if content_list2 is not None:
+            log.info('Check if content is present in output')
+            for item in content_list + content_list2:
+                assert item in file_content
+
+        if content_list is not None:
+            log.info('Check if content is present in output')
+            for item in content_list:
+                assert item in file_content
+
+        if check_value is not None:
+            log.info('Check if value is present in output')
+            assert check_value in file_content
+
+        if check_value_not is not None:
+            log.info('Check if value is not present in output')
+            assert check_value_not not in file_content
+
+        log.info('Reset log file for next test')
+        f.truncate(0)
+
+
+@pytest.fixture(scope="function")
+def create_test_user(topology_st, request):
+    log.info('Create test user')
+    users = nsUserAccounts(topology_st.standalone, DEFAULT_SUFFIX)
+    testuser = users.create_test_user()
+
+    def fin():
+        log.info('Delete test user')
+        if testuser.exists():
+            testuser.delete()
+
+    request.addfinalizer(fin)
+
+
+@pytest.mark.bz1862971
+@pytest.mark.ds4281
+@pytest.mark.skipif(ds_is_older("1.4.2"), reason="Not implemented")
+def test_dsidm_account_entry_status_with_lock(topology_st, set_log_file, create_test_user):
+    """ Test dsidm account entry-status option with account lock/unlock
+
+    :id: d911bbf2-3a65-42a4-ad76-df1114caa396
+    :setup: Standalone instance
+    :steps:
+         1. Create user account
+         2. Run dsidm account entry status
+         3. Run dsidm account lock
+         4. Run dsidm account entry status
+         5. Run dsidm account unlock
+         6. Run dsidm account entry status
+    :expectedresults:
+         1. Success
+         2. Success
+         3. Success
+         4. Success
+         5. Success
+         6. Success
```

Comment:
Probably it makes sense to mention here - what exactly do you expect:

```
state_lock = 'Entry State: directly locked through nsAccountLock'
or
state_unlock= 'Entry State: activated'
```

Reply:
Thanks for pointing that out, I fixed the indentation and messages in expected results, please review.